api/client/kill.go

```diff
@@ -17,17 +18,16 @@ func (cli *DockerCli) CmdKill(args ...string) error {
 
 	cmd.ParseFlags(args, true)
 
-	var errNames []string
+	var errs []string
 	for _, name := range cmd.Args() {
 		if err := cli.client.ContainerKill(name, *signal); err != nil {
-			fmt.Fprintf(cli.err, "%s\n", err)
```

Comment:
Oh I see, you changed "set".  You could do it that way but if people dislike this PR then you'll have to "fix" set().  Guess you're being optimistic. :-)